roles/monitoring_plugins/tasks/main.yml

```diff
@@ -0,0 +1,49 @@
+---
+
+- gather_facts:
+
+- name: Include OS family specific vars
+  include_vars: "{{ ansible_os_family }}.yml"
+
+- name: Include OS distribution/version specific vars
+  include_vars: "{{ lookup('first_found', params) }}"
+  vars:
+    params:
+      files:
+        - "{{ ansible_os_family }}-{{ ansible_distribution }}-{{ ansible_distribution_major_version }}.yml"
+        - "{{ ansible_os_family }}-{{ ansible_distribution }}.yml"
+        - "{{ ansible_os_family }}-{{ ansible_distribution_major_version }}.yml"
+        - default.yml
+      paths:
+        - 'vars'
```

Comment:
You can directly apply the workaround/fix for issue #148

Reply:
A fix for this has been added.

Reply:
The fix would be to change the path location where the include vars should look for variables. 

```
      paths:
        -  "{{role_path}}/vars"
 ```